contrib/macdeploy/gen-sdk

```diff
@@ -99,16 +96,15 @@ def run():
             return tarinfo
         with cd(dir_to_add):
             # recursion already adds entries in sorted order
-            tarfp.add(".", recursive=True, filter=change_tarinfo_base)
+            tarfp.add("./usr/include", recursive=True, filter=change_tarinfo_base)
+            tarfp.add("./usr/lib", recursive=True, filter=change_tarinfo_base)
```

Comment:
This change causes `fatal error: 'CoreFoundation/CoreFoundation.h' file not found` when building the `qt` package in depends.

Reply:
Looks like we have to retain `System/Library/` for Qt. Will return in the next push.